src/test/server_test.cpp

```diff
@@ -266,6 +266,7 @@ TEST(Server, ServerAliveBeforeLoadingModels) {
         if (modelInstance->getStatus().getState() == ovms::ModelVersionState::AVAILABLE)
             break;
     }
+    std::this_thread::sleep_for(std::chrono::milliseconds(10));
```

Comment:
If the issue was that the model couldn't load then I would increase count in L260 x100 or increase timestep in L261 or both instead of increasing whole procedure time from ~50ms to ~60ms.